.github/workflows/release-ci.yml

```diff
@@ -0,0 +1,39 @@
+name: Release CI
+run-name: Release CI triggered from @${{ github.actor }} of ${{ github.head_ref }}
+
+on:
+  workflow_dispatch:
+  merge_group:
+  pull_request:
+  push:
```

Comment:
Would it make since to use
```
on:
  release:
    types: [created, published]
  ```
  instead?  That way you don't need to check for release label either.

Reply:
Changed to that.

My reasoning behind having label that triggers it was for the case where one makes big changes and wants to run more exhaustive testing without creating a release.

Reply:
What you said actually makes more sense rather than triggering on actual release.  What do you think about just having it trigger on `workflow_dispatch` for now?

Reply:
SG, changed to that.